src/main/plugins/org.dita.xhtml/xsl/xslhtml/tables.xsl

```diff
@@ -459,7 +457,7 @@
             <xsl:value-of select="$colspec/@align"/>
           </xsl:when>
           <xsl:otherwise>
-            <xsl:value-of select="$table.align-default"/>
+            <!-- 2.3: Remove default (allow actual default to come from browser or CSS) -->
```

Comment:
Remove this comment and the whole `xsl:otherwise`. We have version control for history.